.envrc

```diff
@@ -20,8 +20,10 @@ reset="$(tput sgr0)"
 #      consequently, using "exit" anywhere will skip this notice from showing.
 #      so need to use set -e, and return 1.
 trap notice ERR
-# If no message is given then we will use this as a default
-error_message="Error without message"
+# This is used to group issues on Sentry.io.
+# If an issue does not call info() or die() it will be grouped under this
+error_message="Unknown issue"
```

Comment:
We should normally not have unknown issues since `advice_` functions or `die` calls should change it.